server/zanata-war/src/test/java/org/zanata/ZanataRestTest.java

```diff
@@ -79,12 +84,11 @@ public final void prepareRestEasyFramework() {
                     providerInstance);
         }
 
-        InMemoryClientExecutor executor =
-                new InMemoryClientExecutor(dispatcher);
```

Comment:
> I think InMemoryClientExecutor is deprecated, isn't it? 

Yes

I've been looking around and can't find anything. There is no test that is using this at the moment (apart from TMXDummyRestTest which has 0 tests in it)